hugo/layouts/people/single.html

```diff
@@ -118,6 +118,11 @@ <h5 class="card-header">Venues</h5>
           </div>
         </div>
       </div>
+
+      <a class="btn btn-warning btn-lg btn-secondary btn-block mb-2" title="Correct problems with the author page" href="https://github.com/acl-org/acl-anthology/issues/new?assignees=anthology-assist&labels=correction%2Cmetadata&projects=&template=02-name-correction.yml&title=Author+Page%3A+{replace+with+author+name}">
```

Comment:
Here we should directly insert the author name into the link. Or, maybe rather the ID, since that is unambiguous here. Let me try that...